static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx

```diff
@@ -9,6 +9,215 @@ import type {
 import {t, tct} from 'sentry/locale';
 import {getInstallConfig as getNodeInstallConfig} from 'sentry/utils/gettingStartedDocs/node';
 
+const getDartConfigureSnippet = (params: DocsParams) => `
+import 'package:sentry/sentry.dart';
+
+Future<void> main() async {
+  await Sentry.init((options) {
+    options.dsn = '${params.dsn}';
+    options.enableMetrics = true;
+  },
+);`;
+
+const getDartVerifySnippet = () => `
+// Add 4 to a counter named "hits"
+Sentry.metrics().increment("hits", value: 4);`;
+
+export const getDartMetricsOnboarding = ({
```

Comment:
As it is only used in dart, I would prefer it to be directly in the dart file.
(same for flutter)